_inc/client/my-plan/checklist-progress-card/index.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import ProgressBar from './progress-bar';
+import QueryChecklistProgress from 'components/data/query-checklist-progress';
```

Comment:
Will `QueryChecklistProgress` return a specific thing when the list is already complete, so we can manipulate (or even hide) the card accordingly, or should that be handled by each component by comparing the number of steps completed to the total?

![image](https://user-images.githubusercontent.com/426388/57454928-43bbed00-726a-11e9-8f32-4d36e0f713a8.png)

Reply:
`QueryChecklistProgress` is a data component that will make sure that data we're interested in is fetched.

Components typically get the data by calling selectors. The most basic selectors just find the data in state and return it, but what you mention is a job that would fit well in a selector: transform checklist data into completion data.

In this PR I've done that inline:

https://github.com/Automattic/jetpack/blob/21108a1084b6d8b68bcb5a466fa9daa7d671934a/_inc/client/my-plan/checklist-progress-card/index.js#L49-L52

That could certainly (and arguably should) be moved (and optimized) in a dedicated selector.

Reply:
> moved (and optimized) in a dedicated selector.

Right, that's kinda what I had in mind; I think that's information anyone building a checklist will need, so we might as well make it available to everyone.

Reply:
Implemented the selector.